Makefile

```diff
@@ -223,18 +223,18 @@ $(LIB_DIR):
 
 $(MEMCACHED_BIN): | $(BIN_DIR) $(LIB_DIR)
 	@echo "Downloading Memcached"
-	curl -L https://archive.org/download/archlinux_pkg_libevent/libevent-2.1.10-1-x86_64.pkg.tar.xz | tar --strip-components=2 --xz -xf - -C $(LIB_DIR) usr/lib
-	curl -L https://archive.org/download/archlinux_pkg_memcached/memcached-1.5.8-1-x86_64.pkg.tar.xz | tar --strip-components=2 --xz -xf - -C $(BIN_DIR) usr/bin/memcached
+	curl -L https://archive.org/download/archlinux_pkg_libevent/libevent-2.1.12-1-x86_64.pkg.tar.zst | tar --strip-components=2 --use-compress-program=unzstd -xf - -C $(LIB_DIR) usr/lib
```

Comment:
ideally we should install from RPM if available

Reply:
where could I get the rpm's?

Reply:
In the CI env, they might be available from Red Hat repositories. You could installing them from dnf if the container image is based on rhel?

Reply:
You can modify the Dockerfile in a way like as this one
```
RUN dnf install -y --setopt=tsflags=nodocs memcached memcached-selinux && \
    dnf clean all && rm -rf /var/cache/*
```

Reply:
I can try again with dnf but my last attempts it was failing due to libraries compatibily

Reply:
But, it is needed Memcached in local to run with the scripts. Even if we installed memcached in the container, we would need another test container to run the test. This is feasible but it will take time, upload the images, change the ci, it is worth?